Dockerfile

```diff
@@ -23,7 +23,7 @@ ARG SONATYPE_WORK="/sonatype-work"
 ARG CONFIG_HOME="/etc/nexus-iq-server"
 ARG LOGS_HOME="/var/log/nexus-iq-server"
 ARG GID=1000
-ARG UID=997
+ARG UID=1000
```

Comment:
If we change the UID I think this will cause a re-occurrence of https://issues.sonatype.org/browse/CLM-17312 ?

After reading through the thread I see these as the relevant points
> the Dockerfile tries to create nexus user with UID 997 but that is already in use by systemd-coredump
> system UIDs should not be used as future changes could try to use the same UID

fwiw 997 and 1000 were chosen because that's the defaults that were assigned when starting the image back when these were set
https://github.com/sonatype/docker-nexus-iq-server/pull/54

I agree with not using system UUIDs, but I think if we update this we will also have to update
https://help.sonatype.com/display/NXIQM/Upgrading+the+IQ+Server

Reply:
Hi @mealingr I agree with your points about updating the doc if we decide to go with this approach.

@troby what Richard is suggesting is what I also suggested on our call. That would be a minimal change on our jenkins file to run with the params that Richard is proposing. So now we have to decide which approach to take that has the least impact. 

@mealingr and @troby based on your experience in the past, when we update these UIDs, do a lot of clients get impacted? Is it too high-risk (as in would it potentially create a lot of support tickets for us?)

Reply:
> @mealingr and @troby based on your experience in the past, when we update these UIDs, do a lot of clients get impacted? Is it too high-risk (as in would it potentially create a lot of support tickets for us?)

The original issue where UID and GID changed affected at least 4 customers
https://issues.sonatype.org/browse/CLM-13092
https://sonatype.zendesk.com/agent/tickets/36272
https://sonatype.zendesk.com/agent/tickets/36250
https://sonatype.zendesk.com/agent/tickets/36503
https://sonatype.zendesk.com/agent/tickets/36532

The next time this issue occurred it was first raised as an internal issue
https://issues.sonatype.org/browse/CLM-17312
One customer raised an issue
https://sonatype.zendesk.com/agent/tickets/48153
but it was resolved by then

So I would say there's a chance we might get some customer issues even if the docs are updated. Although if the docs are updated the issues should be easy to resolve.

Reply:
Updated to 118 in the docs as well:
https://help.sonatype.com/pages/diffpagesbyversion.action?pageId=330916&selectedPageVersions=28&selectedPageVersions=27